tox.ini

```diff
@@ -0,0 +1,67 @@
+[tox]
+skip_missing_interpreters = True
+skipsdist=True
+minversion = 1.8
+envlist =
+    py27-trollius,
+    py32-trollius,
+    py33-trollius,
+    py33-tulip,
+    py34,
+    py35,
+    pypy-trollius,
+    pypy3-trollius,
+    py2-pep8,
+    py3-pep8,
+    packaging
+
+[testenv]
+# This is required in order to get UTF-8 output inside of the subprocesses
+# that our tests use.
+setenv = LC_CTYPE = en_US.UTF-8
+# Pass Display down to have it for the tests available
+passenv = DISPLAY
+whitelist_externals = /bin/bash
+# Install trollius for 2, 3.2, and pypy
+# Install either trollius or tulip for 3.3
+# Asyncio is in the standard library for 3.4+
+deps =
+    nose-cov
+
+    trollius: trollius
+    tulip: asyncio
+
+    # https://github.com/travis-ci/travis-ci/issues/4866
+    py32: coverage<4.0.0
```

Comment:
My guess is that installed directly in travis and it is catch already. If have look more lines down traivs is installing version 3.7